joomla/joomla-cms
# Correct active file highlight in Template Manager

Pull Request for Issue #25395.

### Summary of Changes

Corrects active file detection in file editor.

### Testing Instructions

Create two files in template directory. One file name should contain the file name of another file. E.g. `menu.php` and `newmenu.php`.
Edit `menu.php` in Template Manager.
Check highlighted files on the left.

### Expected result

`menu.php` highlighted.

### Actual result

`menu.php` and `newmenu.php` are highlighted.

### Documentation Changes Required

No.
